chaostoolkit/cli.py

```diff
@@ -3,6 +3,7 @@
 import decimal
 import io
 import json
+import yaml
```

Comment:
sorry to be a pain but this isn't a stdlib module so should go later in the import ordering.